broccoli-template-linter.js

```diff
@@ -101,15 +111,21 @@ TemplateLinter.prototype.processString = function(contents, relativePath) {
   }, this)
         .join('\n');
 
-
-  let output = this._generateTestFile(
-    'TemplateLint - ' + relativePath,
-    [{
-      name: 'should pass TemplateLint',
-      passed: passed,
-      errorMessage: jsStringEscape(relativePath + ' should pass TemplateLint.\n' + errorDisplay)
-    }]
-  );
+  let output = '';
+  if (this._testGenerator) {
```

Comment:
Do we properly support this anyways?  If you have a project today without ember-cli-qunit or ember-cli-mocha (prior to this PR) what happens? What happens after this change?